rust-runtime/aws-smithy-checksums/src/lib.rs

```diff
@@ -220,7 +220,7 @@ impl Checksum for Sha1 {
 }
 
 #[derive(Debug, Default)]
-struct Sha256 {
+pub struct Sha256 {
```

Comment:
I may have missed it but didn't see this used anywhere, why public?

Reply:
This was used in a past version and I forgot to remove `pub` once I stopped needing this.